client/src/components/Home/SearchBar.js

```diff
@@ -283,16 +284,9 @@ export default class SearchBar extends React.Component {
               <option key={item} value={item} />
             ))}
           </datalist>
-          {/* <Link
-            to={{
-              pathname: `/results`,
-              search: `?search=${searchValue}`
-            }}
-          >
-            <button className="search-button" type="submit" disabled={false}>
-              <FontAwesomeIcon icon="search" onClick={this.handleSubmit} />
-            </button>
-          </Link> */}
+          {/* W.8.23.20 QUESTION: how can we change this to handle multiple search terms? or misspellings, alternate capitalization, etc? 
```

Comment:
This is obviously a current limitation.  My thoughts are that we will be able to partially address this in the way we build our queries and by using fuzzy string maching functionality built into Postgres. https://www.freecodecamp.org/news/fuzzy-string-matching-with-postgresql/  

But without tags (which we can discuss with SR) there will still be limitations.  We can / should definitely discuss this more.